exec/java-exec/src/main/java/org/apache/drill/exec/coord/zk/ZKClusterCoordinator.java

```diff
@@ -245,33 +242,29 @@ public DrillbitEndpoint apply(ServiceInstance<DrillbitEndpoint> input) {
         StringBuilder builder = new StringBuilder();
         builder.append("Active drillbit set changed.  Now includes ");
         builder.append(newDrillbitSet.size());
-        builder.append(" total bits.");
-        if (!newDrillbitSet.isEmpty()) {
-          builder.append(" New active drillbits: \n");
-          for (DrillbitEndpoint bit: newDrillbitSet) {
-            builder.append('\t');
-            builder.append(bit.getAddress());
-            builder.append(':');
-            builder.append(bit.getUserPort());
-            builder.append(':');
-            builder.append(bit.getControlPort());
-            builder.append(':');
-            builder.append(bit.getDataPort());
-            builder.append('\n');
-          }
+        builder.append(" total bits. New active drillbits:\n");
```

Comment:
Please change this log from "**New** active drillbits.." to "**Current** active drillbits"